base/shell/explorer/lang/bg-BG.rc

```diff
@@ -36,6 +36,10 @@ BEGIN
         END
         POPUP "&Документи", IDM_DOCUMENTS
         BEGIN
+            MENUITEM "User Profile", IDM_USERPROFILE, MFT_STRING, MFS_ENABLED
```

Comment:
move that to a new menu in shell32 with menu ID 401

Reply:
Menu item ID 401 is already used as `IDM_RUN`.

Reply:
I will do.

Reply:
OK.